dstart

```diff
@@ -10,6 +10,13 @@ fi
 
 set -e
 SERVICE=$1
+
+# these environment variables need to be set differentlty for running the servers on the host machine
+# instead of in the docker container, so fix them up here
+
+export RUNESTONE_PATH=.
+export BOOK_PATH=~/Runestone/books
+
 cd $RUNESTONE_PATH
```

Comment:
@bnmnetp I just wanted to call this out quickly, the lines I added above help me if I always run this script from my root rs folder, but they make this line a no-op. I don't know if you have use cases for running this script when your current working directory isn't the project root, and/or if you have a different way you would prefer to solve this issue of needing different paths in the 2 contexts (I know I'm setting BOOK_PATH to the same location as the default .env file, but I was seeing that somewhere it was trying to resolve it wasn't able to interpret ~ correctly. Doing it this way solves it I assume because bash resolves it and puts the full path in the environment variable).

Reply:
I think it works fine for me because

1. I always run dstart from the rs root.
2. I have both `RUNESTONE_PATH` and `BOOK_PATH` set in my fish config file.